websites/F/Facebook/presence.ts

```diff
@@ -173,6 +173,15 @@ presence.on("UpdateData", async () => {
 		} else if (search && !privacyMode) {
 			presenceData.details = "Watch - Searching for:";
 			presenceData.state = showSeachQuery ? decodeURI(search) : "(Hidden)";
+		} else if (privacyMode)
+			presenceData.details = "Watch - Viewing an user's page";
+		else {
+			presenceData.details = `Watch - Viewing ${
+				document.querySelector('span > a[role="link"] > span').textContent
+			}'s page`;
```

Comment:
I'm not too sure, but the original might overflow on the presence. Maybe this might be better?
```suggestion
			presenceData.details = "Watch";
			presenceData.state = `Viewing ${
				document.querySelector('span > a[role="link"] > span').textContent
			}'s page`
```

Reply:
Oh yes, it can, I'll adapt the code to pass the part after the "Watch" to the state part of the presenceData ! Thanks